force-app/main/default/classes/Testing Recipes/TestDouble.cls

```diff
@@ -0,0 +1,208 @@
+/**
+ * @description Implements an easy and re-usable StubProvider
+ * Utilizes a fluid interface for ease of use.
```

Comment:
`fluid interface` i think we should make it `FluentInterface` so that is consistent with industry (https://martinfowler.com/bliki/FluentInterface.html)

Reply:
I agree. [Fluid](https://blog.hubspot.com/website/fluid-design) is for design.

Reply:
good catch. updated.